exporters/otlp/README.md

```diff
@@ -24,4 +24,4 @@ auto exporter = std::unique_ptr<sdktrace::SpanExporter>(new otlp::OtlpExporter(o
 
 ## Example
 
-For a complete example demonstrating how to use the OTLP exporter, see [examples/otlp](../../examples/otlp).
+For a complete example demonstrating how to use the OTLP exporter, see [examples/otlp](https://github.com/open-telemetry/opentelemetry-cpp/blob/master/examples/otlp/)).
```

Comment:
Same here.

Reply:
It appears markdown files currently only support relative links to files, not folders - is there another way to workaround the absolute URL here?